### ISSUE TITLE
Error with latest tex4ebook and biblatex

### ISSUE THREAD

**Issue by user41974** [michal-h21/tex4ebook#39]
As per the discussion at https://tex.stackexchange.com/questions/408742/error-with-latest-tex4ebook-and-biblatex-using-texlive-2017-windows?noredirect=1#comment1019969_408742 there is an issue with the latest versions of tex4ebook and biblatex. Ulrike Fisher has suggested a fix in the comments.

Thanks.

**Comment by michal-h21** [comment]
Yes I know. This is a tex4ht issue, I will fix it in the sources.
